pennylane/transforms/batch_transform.py

```diff
@@ -64,7 +65,19 @@ def map_batch_transform(
     >>> dev = qml.device("default.qubit", wires=2)
     >>> fn(qml.execute(tapes, dev, qml.gradients.param_shift))
     [array(0.99500417), array(0.8150893)]
+
+    .. warning::
+        qml.transforms.map_batch_transform is deprecated and will be removed in a future release.
+        Instead, a transform can be applied directly to a batch of tapes. See :func:`~.pennylane.transform` for more details.
     """
+
+    warnings.warn(
+        "qml.transforms.map_batch_transform is deprecated."
+        "Instead, a transform can be applied directly to a batch of tapes."
+        "See :func:`~.pennylane.transform` for more details.",

```

Comment:
Note that
````python
:func:`~.pennylane.transform`
````
should not show up in a code-based user warning because it will not link to the docs.